build-system/config.js

```diff
@@ -139,6 +139,7 @@ module.exports = {
     // Files in this testdata dir are machine-generated and are not part
     // of the AMP runtime, so shouldn't be checked.
     '!extensions/amp-a4a/*/test/testdata/*.js',
+    '!examples/*.js',
```

Comment:
Since all JS under `examples/` will not be compiled into our lib, I don't think we need to do those checks. Most of those checks are to restrict the usage of harmful functions in runtime.